h/services/group.py

```diff
@@ -40,9 +40,7 @@ def create_private_group(self, name, userid, description=None):
         group = self._create(name=name,
                              userid=userid,
                              description=description,
-                             joinable_by=JoinableBy.authority,
-                             readable_by=ReadableBy.members,
-                             writeable_by=WriteableBy.members,
+                             access_flags=PrivateGroupMatcher,
```

Comment:
I've punted on coming up with a better name than `access_flags` for now. Currently this only actually contains access flags and to avoid an explosion in the number of named group types there might be an argument for keeping any further properties (e.g. scoped or not scoped) as independent properties outside of the group type definitions in future.